sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -305,13 +308,12 @@ private static TextAppearance getTextAppearance(TextLine textLine) {
                 if (fieldValue != null) {
                     List<FormElement> formElementList = setReferenceElements(fieldValue.getElements(), readResults);
                     FieldData valueData;
-                    // Bounding box, page and text are not returned by the service in two scenarios:
+                    // Bounding box and page an are not returned by the service in two scenarios:
                     //   - When this field is global and not associated with a specific page (e.g. ReceiptType).
                     //   - When this field is a collection, such as a list or dictionary.
                     //
                     // In these scenarios we do not set a ValueData.
-                    if (fieldValue.getText() == null && fieldValue.getPage() == null
-                        && CoreUtils.isNullOrEmpty(fieldValue.getBoundingBox())) {
+                    if (fieldValue.getPage() == null && CoreUtils.isNullOrEmpty(fieldValue.getBoundingBox())) {
```

Comment:
if fieldValue.getText() == null, do we create an null value FieldData and possible having BoundingBox?

Reply:
Yes, we create a FieldData with null text but values for page and BoundingBox.